<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZarzadzaniePrzegladami</class>
 <widget class="QWidget" name="ZarzadzaniePrzegladami">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTableView" name="tablica"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Przeglad_Label">
         <property name="text">
          <string>Id Przeglądu</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Przeglad_Line"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Data_Label">
         <property name="text">
          <string>Data Wykonania</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="Data_Edit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="Do_Label">
         <property name="text">
          <string>Do daty</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="Do_Edit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="Samochod_Label">
         <property name="text">
          <string>Id Samochodu</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Samochod_Box"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Wroc_Button">
       <property name="text">
        <string>Wróć</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Usun_Button">
       <property name="text">
        <string>Usuń</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Dodaj_Button">
       <property name="text">
        <string>Dodaj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
